docs/README.md

```diff
@@ -1,3 +1,201 @@
-# Documentation
+# .NET Swift interop tooling documentation
 
-This document outlines the functional design of the tooling and projections between Swift and .NET.
\ No newline at end of file
+This document provides a detailed overview of the .NET Swift interop tooling, focusing on the projections between Swift and .NET, and the functional design of the tooling.
```

Comment:
Might be worth explicit stating these tools will work with C# only and any other .NET language is a non-goal.

Reply:
Added.